dsb-integration/src/test/java/com/spectralogic/dsbrowser/integration/tasks/Ds3PutJobTest.java

```diff
@@ -103,14 +101,18 @@ public static void setConnection() {
             session = CreateConnectionTask.createConnection(SessionModelService.setSessionModel(savedSession, false), resourceBundle, buildInfoService);
 
             final Path path;
+            ImmutableList<Pair<String,Path>> pair;
```

Comment:
I'm confused why this is declared here.  I don't see where else this is used after being assigned in the `try-catch`